pkgs/development/libraries/relibc/default.nix

```diff
@@ -73,6 +73,6 @@ redoxRustPlatform.buildRustPackage rec {
     description = "C Library in Rust for Redox and Linux";
     license = licenses.mit;
     maintainers = [ maintainers.aaronjanse ];
-    platforms = platforms.redox;
+    platforms = platforms.redox ++ [ "x86_64-linux" ];
```

Comment:
Upstream claims to support aarch64-linux too

Reply:
I tried but there is some issue with the vendoring on aarch64.

Reply:
Ah ok, thanks for giving it a try.